### ISSUE TITLE
Subtitles of wrong group being downloaded

### ISSUE THREAD

**Issue by hotio** [BenjV/autosub#7]
It seems that a lot of subtitles from a different group are being downloaded...

Using the following settings:
![naamloos](https://cloud.githubusercontent.com/assets/26902309/26284972/ae14020a-3e46-11e7-926c-27a2f02302da.png)

Result:
![naamloos](https://cloud.githubusercontent.com/assets/26902309/26284995/f61e93bc-3e46-11e7-81d9-806c6ec71155.png)
File: DEFLATE
Downloaded: NTB
![naamloos](https://cloud.githubusercontent.com/assets/26902309/26285018/942c80fa-3e47-11e7-8632-20b163ac1c79.png)
File: NTB
Downloaded: CTRLHD

Sometimes, they are in sync, mostly not... I would assume if group is checked, the group should be the same?

**Comment by BenjV** [comment]
Please post the exact names of the videofiles so I can test this.

**Comment by hotio** [comment]
```
Billions - S02E08 - billions.s02e08.1080p.webrip.x264-deflate.mkv
Better Call Saul - S01E03 - better.call.saul.s01e03.webdl-720p.ac3.5.1.x264-ntb.mkv
```

**Comment by BenjV** [comment]
I found a little bug in the score matching which caused this issue.
I corrected it and a new version (1.1.4) is on GitHub.
